_source/logzio_collections/_log-sources/aws-control-tower.md

```diff
@@ -0,0 +1,204 @@
+| `ca-central-1`   | [![Deploy to AWS](https://dytvr9ot2sszz.cloudfront.net/logz-docs/lights/LightS-button.png)](https://console.aws.amazon.com/cloudformation/home?region=ca-central-1#/stacks/create/review?templateURL=https://logzio-aws-integrations-ca-central-1.s3.amazonaws.com/s3-hook/control-tower/0.0.1/sam-template.yaml&stackName=logzio-control-tower&param_logzioToken=<<LOG-SHIPPING-TOKEN>>&param_logzioListener=https://<<LISTENER-HOST>>:8071)     | 
+
+
+##### Specify stack details
+
+Specify the stack details as per the table below, check the checkboxes, and select **Create stack**.
+
+| Parameter   | Description                                                                                           | Required/Default |
+|-------------|-------------------------------------------------------------------------------------------------------|------------------|
+| `logLevel`  | Log level for the Lambda function. Can be one of: `debug`, `info`, `warn`, `error`, `fatal`, `panic`. | Default: `info`  |
+| `s3HookArn` | The ARN of your S3 Hook Lambda function.                                                              | **Required**     |
+
+
+<!-- info-box-start:info -->
+It can take a few minutes after the stack creation for EventBridge rule to be triggered.
+{:.info-box.note}
+<!-- info-box-end -->
+
+<!-- info-box-start:info -->
+If want to delete the S3 Hook Stack - you'll need to detach the policy "LambdaAccessBuckets" first.
+{:.info-box.note}
+<!-- info-box-end -->
+
+##### Check Logz.io for your logs
+
+Give your logs some time to get from your system to ours, and then open [Kibana](https://app.logz.io/#/dashboard/kibana).
+
+If you still don't see your logs, see [log shipping troubleshooting]({{site.baseurl}}/user-guide/log-shipping/log-shipping-troubleshooting.html).
+
+
+
+</div>
+</div>
+<!-- tab:end -->
+
+<!-- tab:start -->
+<div id="Advanced">
+
+#### Advanced settings
+
+##### Filtering files
+
+If there are specific paths within the bucket that you want to pull logs from, you can use the `pathsRegex` variable.
+This variable should hold a comma-seperated list of regexes that match the paths you wish to extract logs from.
+**Note**: This will still trigger your Lambda function every time a new object is added to your bucket. However, if the key does not match the regexes, the function will quit and won't send the logs.
+
+
+##### Adding object path as logs field
+
+In case you want to use your objects' path as extra fields in your logs, you can do so by using `pathToFields`.
+
+For example, if your objects are under the path: `oi-3rfEFA4/AWSLogs/2378194514/file.log`, where `oi-3rfEFA4` is org id, `AWSLogs` is aws type, and `2378194514` is account id. 
+
+Setting `pathToFields` with the value: `org-id/aws-type/account-id` will add to logs the following fields:
+`org-id`: `oi-3rfEFA4`, `aws-type`: `AWSLogs`, `account-id`: `2378194514`.
```

Comment:
```suggestion
Setting `pathToFields` with the value: `org-id/aws-type/account-id` will add to logs the following fields:
`org-id: oi-3rfEFA4`, `aws-type: AWSLogs`, `account-id: 2378194514`.
```

I think maybe this will look better?